spec/rubocop/cop/style/concat_array_literals_spec.rb

```diff
@@ -37,7 +37,27 @@
   it 'registers an offense when using `concat` with single element `%i` array literal argument' do
     expect_offense(<<~RUBY)
       item.concat(%i[item])
-           ^^^^^^^^^^^^^^^^ Use `push` with elements as arguments without array brackets instead of `concat(%i[item])`.
+           ^^^^^^^^^^^^^^^^ Use `push(:item)` instead of `concat(%i[item])`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      item.push(:item)
```

Comment:
minor -- I find it confusing that this test file uses `item` for both the list and the item being inserted. `list.push(:item)` would make more sense

Reply:
Agreed, but this is how this is done for this entire spec file. Do not want to change the lines history and make diff bigger than is needed.
But if requested, will do with no problems..

Reply:
Feel free to add a second commit that changes this. I'd suggest using something like `arr` instead of `item`.

Reply:
Updated.